src/components/LHNOptionsList/OptionRowLHN.tsx

```diff
@@ -238,8 +237,10 @@ function OptionRowLHN({reportID, isFocused = false, onSelectRow = () => {}, opti
                                             (optionItem.shouldShowSubscript ? (
                                                 <SubscriptAvatar
                                                     backgroundColor={hovered && !isFocused ? hoveredBackgroundColor : subscriptAvatarBorderColor}
-                                                    mainAvatar={optionItem.icons[0]}
-                                                    secondaryAvatar={optionItem.icons[1]}
+                                                    // because of the check above we know that there is an icon on index 0
+                                                    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                                                    mainAvatar={optionItem.icons.at(0)!}
```

Comment:
Maybe we can check specifically for this item as well to get rid of null assertion and lint disable.
So, in this file, it can be:
```
const firstIcon = optionItem.icons?.at(0);
....
{!!optionItem.icons?.length && firstIcon &&
....
mainAvatar={firstIcon}
```
Just make sure that there is no regressions. 
Also, maybe @blazejkustra has some suggestions?

Reply:
Let's do it this way, I like it 👍